srcs/signals.c

```diff
@@ -41,17 +41,15 @@ static void	signal_handler_child(int sig_num)
 	return ;
 }
 
-void	set_signals(t_status status)
+void	set_signals(t_status status, t_mode	mode)
 {
-	if (status == CHILD_PROCESS)
+	if (status == CHILD_PROCESS && mode == INTERACTIVE)
 	{
-		DEBUG(printf("current_signal = %d\n", status);)
 		signal(SIGINT, signal_handler_child);

```

Comment:
So shouldn't we print ^C manually, now that we are using termcaps? (couldn't comment on the correct line as it's a piece of code that didn't change and was condensed)

Reply:
nope. Actually I checked in bash, the ^C never gets printed even in the child process I think on mac. :o
What does it do on your computer ? 
I can add that easily if I'm wrong.

Reply:
<img width="171" alt="Captura de ecrã 2022-05-02, às 23 13 13" src="https://user-images.githubusercontent.com/63653190/166335689-07a7b3b8-4ee1-4df6-95b4-4918378372b9.png">
in mine it prints. in yours it doesn't?

Reply:
on my computer it does. On the mac earlier it didn't that's why I removed it earlier. Let's check tomorrow.